BookingsApi/BookingsApi.DAL/Queries/GetHearingByIdQuery.cs

```diff
@@ -42,7 +42,8 @@ public async Task<VideoHearing> Handle(GetHearingByIdQuery query)
                 .Include(x => x.HearingType)
                 .Include(x => x.HearingVenue)
                 .Include(x => x.Endpoints).ThenInclude(x => x.DefenceAdvocate).ThenInclude(x => x.Person)
-                .Include(x => x.Allocations).ThenInclude(x => x.JusticeUser)
+                .Include(x => x.Allocations)// .ThenInclude(x => x.JusticeUser)
```

Comment:
Shouldn't need to change this, `.Include(x => x.Allocations).ThenInclude(x => x.JusticeUser)` should work

Reply:
done

Reply:
this isn't done

Reply:
done